SAMv1.tex

```diff
@@ -1204,6 +1204,48 @@ \subsection{C source code for computing bin number and overlapping bins}\label{s
 \end{verbatim}
 }
 
+\subsection{Splitting BAM}\label{sec:code}
+A BAM file can be processed in parallel by conceptually dividing the file into
+splits (typically of a fixed, but arbitrary, number of bytes) and for each
+split processing alignments from the first known alignment after the split
```

Comment:
We might want a sentence here describing why this additional index is necessary and this use can't be handled by the bai.